pkgs/data/themes/nordic/default.nix

```diff
@@ -2,7 +2,7 @@
 
 stdenv.mkDerivation rec {
   pname = "nordic";
-  version = "unstable-2021-11-19";
+  version = "2.1.0";
```

Comment:
Shouldn't that change the revs below?

Reply:
No. Because the revision currently used in the nix expression is the same that was tagged as v2.1.0 in the git repository, two days after the publication of the revision.

The author organizes the themes using git branches. The master branch corresponds to the base nordic theme. The theme variants are available at other branches. Therefore in order to keep offering all of them we still need to fetch the specific revisions in each branch.